docs/api/websockets.md

```diff
@@ -117,8 +117,8 @@ type WebSocketData = {
 // TypeScript: specify the type of `data`
 Bun.serve<WebSocketData>({
   fetch(req, server) {
-    // use a library to parse cookies
-    const cookies = parseCookies(req.headers.get("Cookie"));
+    const cookies = new Bun.CookieMap(req.headers.get("cookie")!);
+
     server.upgrade(req, {
       // this object must conform to WebSocketData
       data: {
```

Comment:
`cookies["X-Token"],` below needs to be changed to `cookies.get("X-Token")`

Reply:
Thank you @pfgithub